recipe/gen_patch_json.py

```diff
@@ -1315,6 +1314,10 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             deps.append("numba !=0.54.0")
             record["depends"] = deps
 
+        if (record_name, record["version"], record["build"]) == ("jax", "0.4.14", "pyhd8ed1ab_0"):
+            deps = record.get("depends", [])
+            deps[deps.index("python >=3.8")] = "python >=3.9"
```

Comment:
Did you want to patch the numpy lower bound too?

Reply:
The numpy lower bound is correct with >=1.22